sdk/keyvault/azure-security-keyvault-keys/src/samples/java/com/azure/security/keyvault/keys/KeyAsyncClientJavaDocCodeSnippets.java

```diff
@@ -145,6 +147,32 @@ public void deleteKeySnippets() {
         // END: com.azure.security.keyvault.keys.async.keyclient.deleteKey#string
     }
 
+    /**
+     * Generates a code sample for using {@link KeyAsyncClient#importKey(String, JsonWebKey)}
+     */
+    public void importKeySnippets() {
+        KeyAsyncClient keyAsyncClient = createAsyncClient();
+        JsonWebKey jsonWebKeyToImport = new JsonWebKey();
+        // BEGIN: com.azure.security.keyvault.keys.keyasyncclient.importKey#string-jsonwebkey
+        keyAsyncClient.importKey("keyName", jsonWebKeyToImport).subscribe(keyResponse ->
+            System.out.printf("Key is imported with name %s and id %s \n", keyResponse.getName(), keyResponse.getId()));
+        // END: com.azure.security.keyvault.keys.keyasyncclient.importKey#string-jsonwebkey
+
+        // BEGIN: com.azure.security.keyvault.keys.keyasyncclient.importKey#options
+        ImportKeyOptions options = new ImportKeyOptions("keyName", jsonWebKeyToImport)
+            .setHardwareProtected(false);
+        keyAsyncClient.importKey(options).subscribe(keyResponse ->
+            System.out.printf("Key is imported with name %s and id %s \n", keyResponse.getName(), keyResponse.getId()));
+        // END: com.azure.security.keyvault.keys.keyasyncclient.importKey#options
+
+        // BEGIN: com.azure.security.keyvault.keys.keyasyncclient.importKeyWithResponse#options-response
+        ImportKeyOptions importKeyOptions = new ImportKeyOptions("keyName", jsonWebKeyToImport)
+            .setHardwareProtected(false);
+        keyAsyncClient.importKey(importKeyOptions).subscribe(keyResponse ->
```

Comment:
should use `importKeyWithResponse` here.

Reply:
good catch, updated.